index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link rel="stylesheet" href="style.css">
+    <link
+    rel="stylesheet"
+    href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>BankJoy</title>
+      <div class="contact">
+        <span><i class="fa fa-phone" aria-hidden="true"></i>714-316-8395</span>
+        <span><i class="fa fa-envelope-o" aria-hidden="true"></i>
+          <a href="mailto:contact@email.com">Email Us</a></span>
+      </div>
```

Comment:
Looks like you've got everything in the `head` tag. The content should be in a `body` tag.